### PR TITLE
Add brotli fuzzer

### PR DESCRIPTION
**Pull request by eustas**

### CODE REVIEW

File: projects/brotli/project.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+homepage: "https://github.com/google/brotli"
```
Comment (inferno-chromium) [inline projects/brotli/project.yaml]: Need to add yourself in primary_contact field ?

File: projects/brotli/brotli_fuzzer.cc [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```
Comment (inferno-chromium) [inline projects/brotli/brotli_fuzzer.cc]: Please move the fuzzer code in brotli repo, we want fuzz target there. See https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md. Then you you don't need to copy it here.

File: projects/brotli/project.yaml (hunk 1 above)
Comment (eustas) [inline projects/brotli/project.yaml]: Done

**Comment by inferno-chromium** [comment]
Awesome, thanks LGTM and merged.

**Pull request merged**